### PR TITLE
Going the distance

### PR DESCRIPTION
**Pull request by cchriste**

### CODE REVIEW

**Comment by archanasri** [comment]
@sheryjoe This PR resolves #810. The issue mentioned that this also resolves fids-related executables. So can we close #1020, #1010 and #1011?

**Comment by cchriste** [comment]
Ready except for Windows build failing. Working on it today.

**Comment by cchriste** [comment]
Failing test works fine on my local system after fresh install of everything, but unsure if it was working before due to unrelated problems. Could caching of actions play some role in the use case hanging?

**Comment by akenmorris** [comment]
I've removed the windows cache files and restarted the job.

**Comment by cchriste** [comment]
Notebooks verified. This branch can now be merged.

File: install_shapeworks.bat [hunk 1]
```diff
@@ -59,6 +59,8 @@ call .\Installation\conda_env_setup.bat
 REM installs for jupyter notebooks
 call pip install nbstripout==0.5.0
 call pip install pyvista==0.30.1
+call pip install pyqt5==5.15.6
+call pip install pyvistaqt==0.6.0
```
Comment (akenmorris) [inline install_shapeworks.bat:63]: Should we remove these to match?

**Comment by cchriste** [comment]
Yes, thanks for noticing!

**Review by akenmorris — APPROVED** [review]

**Pull request merged**